conda/recipes/cuml/meta.yaml

```diff
@@ -51,7 +51,7 @@ requirements:
     - libcuml={{ version }}
     - libcumlprims {{ minor_version }}
     - pyraft {{ minor_version }}
-    - cupy>=7.8.0,<11.0.0a0
+    - cupy>=7.8.0,<12.0.0a0
```

Comment:
Small question, should we do the same for the dev environments?

Reply:
We can do that. Looked at the environment files here, but didn't see `cupy` listed. Do we want to add it as part of this PR?

Reply:
I think currently it is tied to the meta packages or brought by cudf, so we should be fine with the PR as is

Reply:
Sounds good. Happy to follow up if needed :)